presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/PrestoSparkNativeExecutionShuffleManager.java

```diff
@@ -143,10 +155,12 @@ public Iterator<Product2<K, V>> read()
             extends ShuffleWriter<K, V>
     {
         private final long[] mapStatus;
+        private static final long ONE = 1L;
```

Comment:
Can we name this constant, instead of ONE. If no name exists, let us remove it and directly leave 1L.

Reply:
Changed it to DEFAULT_MAP_STATUS